Documentation/DocuBlocks/Rest/Collections/delete_api_collection.md

```diff
@@ -4,6 +4,12 @@
 
 @RESTHEADER{DELETE /_api/collection/{collection-name}, Drops a collection}
 
+@HINTS
+{% hint 'warning' %}
```

Comment:
shouldn't these then be the comments? or do you change that on the way into the swagger json?

Reply:
That is not really possible. The hint type is taken from the Gitbook hint tag, and there can be multiple hint boxes in the body of `@HINTS`. The hint type is prepended to each hint body:

```
@HINTS
There can be plain text here.

{% hint 'danger' %}
Serious warning!
{% endhint %}

{% hint 'info' %}
Something useful.
{% endhint %}
```

becomes:

There can be plain text here.
**Danger:** Serious warning!
**Info:** Something useful.